src/components/forms/SignInForm.js

```diff
@@ -53,11 +53,10 @@ class SigninForm extends Component {
             value={password}
             onChange={e => this.onInputChange("password", e)}
           />
-          <h3>Remember Me</h3>
           <input type="submit" value="Sign In" />
         </form>
-        <h3>Forgot Password?</h3>
         <h3>Don't Have An Account?</h3>
+        <a href="/register">Sign In</a>
```

Comment:
I'll style it as a button later. I'll try to style both the register form & sign in form today.
Sorry this one should be as the following. I'll fix it when I do the styling.
<a href="/register">Register Account</a>